codalab/model/bundle_model.py

```diff
@@ -984,10 +984,6 @@ def transition_bundle_finalizing(self, bundle, user_id, worker_run, connection):
         bundle_update = {'state': State.FINALIZING, 'metadata': metadata}
 
         self.update_bundle(bundle, bundle_update, connection)
-
-        if user_id == self.root_user_id:
```

Comment:
Not exactly sure why we compare the `user_id == self.root_user_id`? During my testing, I saw they are equal all the time. Maybe I missed some non-trivial details here?

Reply:
With user-owned workers, is it different?

Reply:
I think this is here to only charge users for non user-owned workers.  This is important, probably should add a comment.

Reply:
I tested using user-owned worker, but still always got them equal. If we want to avoid adding time spent on user-owned workers, we need to find another way.

Reply:
Yes. Time used in user-owned workers should be excluded from this calculation. I have updated the code to increment time used only on public workers